examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -234,8 +243,8 @@ export class TaskList extends DataObject implements ITaskList {
             }
         });
 
-        for (const [id, taskData] of this.root) {
-            const typedTaskData = taskData as PersistedTask;
+        for (const [id, task] of this._draftData.entries()) {
```

Comment:
Did you determine if the `.entries()` is required after fixing the handle storage?  If not required, prefer to leave it out.

This also looks like it's not hitting some expected code checks, since `this._draftData` can be undefined.  This would be a good place to use the private get helper:

```suggestion
        for (const [id, task] of this.draftData) {
```